docs/reference/mapping/removal_of_types.asciidoc

```diff
@@ -320,7 +320,7 @@ PUT tweets
     "index.mapping.single_type": true
   },
   "mappings": {
```

Comment:
In line 242 above, under changes coming in 6.x, mention addition of `_doc` as an acceptable type name to ease the transition to new URLs